### DISCUSSION TITLE
Is there anyway to add padding to the last line of the code

### DISCUSSION THREAD

**Discussion by nibin-matteo-git** [tajmone/Sublime-PML#23] [Q&A] [answered]
The horizontal scroll bar is overlapping with the last line of the code. Tried using the scroll past end but I didnt like it as it was not sorta a pain when scrolling to the end of the file.
There was an option to increase the bottom padding but it increases the bottom padding for every line.
Is there anyway to increase the padding for the last line alone.
<img width="715" alt="Screenshot 2021-05-28 at 9 56 14 AM" src="https://user-images.githubusercontent.com/80332406/119929416-14848f80-bf9b-11eb-8116-7e965398d3db.png">

**Comment by tajmone** [comment, marked as answer]
> Is there anyway to increase the padding for the last line alone.

I'm not aware of any such option. Also, this week Sublime Text updated to version 4, and I hadn't time to check all the new options that were added.

Usually I configure my projects' [EditorConfig] settings to ensure that all source files have an empty line at the end of the file:

```
insert_final_newline = true
```

with that setting in place, Sublime Text will automatically add the empty line at save time, which solves the problem altogether. Adding an empty line at the end of a source file is considered good practice, unless the file format doesn't allow it. For PML projects, we always add the final newline.

As for Sublime Text settings, you can always open the __Preferences__ » __Settings__ menu, which will open both the default and user settings in a new editor instance. The default settings file contains all the available options, either commented out or with actually settings, and usually with an explicatory comment too. If there isn't a dedicated option there then there probably isn't one (which I suspect is the case, since all I can see are settings for line spacing, but no page margin settings).




[EditorConfig]: https://editorconfig.org
